website/util/sanitize.py

```diff
@@ -78,7 +78,7 @@ def _ensure_clean(value):
 
 
 # TODO: Remove unescape_entities when mako html safe comes in
-def unescape_entities(value):
+def unescape_entities(value, safe=None):
```

Comment:
Can you confirm that no result of filenames unescaped using this parameter is *ever* shown to the user / passed to the front end?

See comments on this ticket about bleach for context on why we've so carefully avoided adding this functionality. If the output of this version of the function is ever sent back to the front end, it could be problematic.
 https://openscience.atlassian.net/browse/EOSF-202?jql=project%20%3D%20EOSF%20AND%20text%20~%20ampersand 

Because it's such a common and regrettable sticking point, I highly recommend adding a description of this parameter (and a warning not to use it freely) to the docstring for this function.